mne/bem.py

```diff
@@ -2187,3 +2187,59 @@ def check_seghead(surf_path=op.join(subj_path, 'surf')):
             incomplete=incomplete, extra=msg)
         write_bem_surfaces(dec_fname, dec_surf, overwrite=overwrite)
     logger.info('[done]')
+
+
+def distance_to_bem(
+    pos: np.array, bem: ConductorModel, trans: Transform = None
+) -> np.array:
+    """Calculate the distance of postions to inner skull surface.
+
+    Parameters
+    ----------
+    pos : array, shape = (3,) | (n, 3)
+        Position(s) in m, in head coordinates.
+    bem : instance of ConductorModel
+        Conductor model.
+    trans : array, shape = (4, 4) | instance of Transform
+        Transform matrix.
+    
+    Returns
+    -------
+    distances : array, shape = (3,) | (n, 3)
+    """
+
+    if pos.size == 3:
+        single_pos = True
+        pos = np.expand_dims(pos, axis=0)
+    else:
+        single_pos = False
+
+    n = pos.shape[0]
+
+    distance = np.zeros((n,))
+
+    if bem["is_sphere"]:
+        center = bem["r0"]
+
+        if trans:
+            center = apply_trans(trans, center, move=True)
+        radius = bem["layers"][0]["rad"]
+
+        for i in range(n):
+            distance[i] = radius - np.linalg.norm(pos[i, :] - center)
```

Comment:
something like this:

`distance = radius - np.linalg.norm(pos - center, axis=1)`

should work and avoid the for loop

Reply:
Yes this was leftover from trying to be (too) clever and combine spheres and BEMs. Done.